torch/distributed/run.py

```diff
@@ -713,6 +713,10 @@ def config_from_args(args) -> Tuple[LaunchConfig, Union[Callable, str], List[str
         # This env variable will be passed down to the subprocesses
         os.environ["OMP_NUM_THREADS"] = str(omp_num_threads)
 
+    log_line_prefix_template = "[${role}:${local_rank}]"
```

Comment:
I'm confused at what the default prefix format should be. Is it
`[trainer1]:` or `[trainer:1]`?

If the former, shouldn't this be `[${role}${local_rank}]:`? Otherwise this would prefix `[trainer:1]<log msg>` instead of `[trainer1]:<log msg>`